app/views/exercises/_editor_output.html.slim

```diff
@@ -59,15 +59,10 @@ div.d-grid id='output_sidebar_uncollapsed' class='d-none col-sm-12 enforce-botto
         .progress
           .progress-bar role='progressbar'
 
-      br
-      - if lti_outcome_service?(@exercise.id, external_user_id)
-        p.text-center = render('editor_button', classes: 'btn-lg btn-success d-none', data: {:'data-url' => submit_exercise_path(@exercise)}, icon: 'fa-solid fa-paper-plane', id: 'submit', label: t('exercises.editor.submit'))
         - if @exercise.submission_deadline.present? || @exercise.late_submission_deadline.present?
           #deadline data-submission-deadline=@exercise.submission_deadline&.rfc2822 data-late-submission-deadline=@exercise.late_submission_deadline&.rfc2822
             h4 = t('exercises.editor.deadline')
             = t('exercises.editor.hints.disclaimer')
```

Comment:
I might be wrong, but I would have expected that the indention level of these lines should change (move more to the left). Have you verified everything is working here correctly?

Reply:
It looks the same when moving it two spaces left. But I don't know when the expression is true. When I set a submission deadline for the exercise on openHPI it is not shown.

Reply:
You need to specify a submission deadline within the exercise on CodeOcean -- not on openHPI. This is completely independent of any LTI information.

When reading the code (but without trying live), I have the feeling that the deadline information is nested under the `<div id='score'>`, which should not be the case. Moving those lines to the left would have them on the same level as the `<div id='score'>`, not nested below.